extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -166,6 +169,27 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyala-player'], fu
 				}
 			});
 
+			player.mb.subscribe( OO.EVENTS.PLAYHEAD_TIME_CHANGED, 'featured-video', function(eventName, time, totalTime) {
+				var secondsPlayed = Math.floor(time),
+					percentage = Math.round(time / totalTime * 100);
+
+				if ( secondsPlayed % 5 === 0 && secondsPlayed !== playTime ) {
```

Comment:
In the proposal of drop off tracking I proposed to stop sending tracking events when user uses seek bar (look at examples here https://wikia-inc.atlassian.net/wiki/display/PREMIUM/Videos+drop-offs+tracking)